psi4/src/psi4/libmints/dimension.h

```diff
@@ -44,14 +44,10 @@ class PSI_API Dimension {
 
    public:
     Dimension();
-    Dimension(const Dimension& other);
```

Comment:
guess it's not used at present, but this is ordinary copy constructor, so by rule of three should keep this and copy assignment. or maybe since destructor is empty should kill off all three?

Reply:
I think I killed off all non-trivial ones?

Reply:
OK, I guess you learn something new every day. Explicit destructor removed.